pymc3/sampling.py

```diff
@@ -432,7 +438,10 @@ def init_nuts(init='advi', n_init=500000, model=None, **kwargs):
 
     if init == 'advi':
         v_params = pm.variational.advi(n=n_init)
-        start = pm.variational.sample_vp(v_params, 1, progressbar=False, hide_transformed=False)[0]
+        if njobs > 1:
+            start = pm.variational.sample_vp(v_params, njobs, progressbar=False, hide_transformed=False)
+        else:
+            start = v_params.means
```

Comment:
I think we always want a sample from the posterior to start in the "typical set". The mean can be far away from that, which is counter-intuitive, but true for high-dimensional models: https://www.youtube.com/watch?v=pHsuIaPbNbY